src/seedsigner/gui/screens/seed_screens.py

```diff
@@ -427,6 +427,21 @@ def __post_init__(self):
         )
         self.components.append(self.fingerprint_icontl)
 
+class SeedSwitchElectrumModeScreen(ButtonListScreen):
+    button_data: list = None
+
+    def __post_init__(self):
+        self.show_back_button = False
+        self.title = "Switch to Electrum?"
+        self.is_bottom_list: bool = True
```

Comment:
"Switch" isn't quite the right verb here. 

Would "Electrum Seed Detected" fit in the title? If not, the title can just say "Electrum Seed" since the body text will have more info.

Reply:
Same response as below--agree on removing switch language but it is in a tiny doubt if the user meant to enter BIP-39 or electrum.  Alternatively we could check if it meets both checksums and only have a confirmation screen in that edge case, and go with the Electrum Seed declaration and warning if it is a valid electrum seed but not a valid BIP-39 checksum.  Does that sound good?  Confirmation screen if it meets both checks, warning screen if it's electrum-only

Reply:
Yes, It's making more sense to me too, especially after learning the collisions are more often than I had initially thought.  And in this way it keeps the BIP-39 flow completely unaffected by default even when they happen to collide with a valid Electrum seed (apparently 9/2048 chance which is much higher than I had calculated).
This also removes the need for a new screen altogether possibly, at least during seed entry